pkg/kubeconfig/kubeconfig.go

```
@@ -18,75 +18,78 @@ package kubeconfig
 
-func LoadAndMerge(ctx context.Context, client kubernetes.Clientset, name, controlPlaneType string) error {
-	// TODO add a kubeconfig parameter
-	konfig, err := LoadKubeconfig("")
+func unMarshallCM(obj runtime.Object) (*corev1.ConfigMap, error) {
+	jsonData, err := json.Marshal(obj)
 	if err != nil {
-		return err
+		return nil, fmt.Errorf("error marshaling object %s", err)
 	}
+	cm := corev1.ConfigMap{}
+	json.Unmarshal(jsonData, &cm)
```

golangci-lint (1.64.8)

Error return value of `json.Unmarshal` is not checked

(errcheck)